scipy/stats/_distn_infrastructure.py

```diff
@@ -2560,7 +2557,7 @@ def fit(self, data, *args, **kwds):
         ------
         TypeError, ValueError
             If an input is invalid
-        FitError
+        `~scipy.stats._warnings_errors.FitError`
```

Comment:
This is rendering as expected:
![image](https://user-images.githubusercontent.com/6570539/161403081-1027b6ed-049e-4fc0-bbb3-27a42715c75e.png)
![image](https://user-images.githubusercontent.com/6570539/161403086-37255a4a-9ca3-4610-8f0d-eb86277c3cb1.png)

Reply:
Well, almost. I was hoping they would identify their superclass.